neo/IO/Caching/FIFOSet.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Specialized;
+
+namespace Neo.IO.Caching
+{
+    internal class FIFOSet<T> where T : IEquatable<T>
+    {
+        private readonly int maxCapacity;
+        private readonly int removeCount;
+        private readonly OrderedDictionary data;
+
+        public FIFOSet(int maxCapacity, decimal? batchSize = 0.1m)
+        {
+            if (maxCapacity <= 0) throw new ArgumentOutOfRangeException(nameof(maxCapacity));
+            if (batchSize <= 0) throw new ArgumentOutOfRangeException(nameof(batchSize));
+
+            this.maxCapacity = maxCapacity;
+            this.removeCount = batchSize != null ? (int)(batchSize <= 1.0m ? maxCapacity * batchSize : maxCapacity) : 1;

```

Comment:
I think `null` should be the same as the default value.

Reply:
Done!